lib/ansible/modules/web_infrastructure/letsencrypt_account.py

```diff
@@ -0,0 +1,250 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2016 Michael Gruener <michael.gruener@chaosmoon.net>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: letsencrypt_account
+author: "Felix Fontein (@felixfontein)"
+version_added: "2.6"
+short_description: Create, modify or delete accounts with Let's Encrypt
+description:
+   - "Allows to create, modify or delete accounts with Let's Encrypt.
+      Let's Encrypt is a free, automated, and open certificate authority
+      (CA), run for the public's benefit. For details see U(https://letsencrypt.org)."
+   - "This module only works with the ACME v2 protocol."
+extends_documentation_fragment:
+  - letsencrypt
+options:
+  state:
+    description:
+      - "The state of the account, to be identified by its account key."
+      - "If the state is C(absent), the account will either not exist or be
+         deactivated."
+      - "If the state is C(changed_key), the account must exist."
+    required: true
+    choices:
+    - present
+    - absent
+    - changed_key
+    version_added: "2.5"
```

Comment:
Please remove the version_added here because module is new.

Reply:
I guess that was a copy'n'paste error.